src/firefly/js/core/background/BgMaskPanel.jsx

```diff
@@ -52,10 +48,10 @@ export const BgMaskPanel = React.memo(({componentKey, onMaskComplete, style={}})
 
     const msg = jobInfo?.errorSummary?.message || jobInfo?.jobInfo?.progressDesc || 'Working...';
 
-    const Options = () => ( (inProgress || true) &&
+    const Options = () => (
         <Stack direction='row' spacing={1}>
-            <Button variant='solid' color='primary' onClick={sendToBg}>Send to background</Button>
-            <Button onClick={abort}>Cancel</Button>
+            <Button variant='solid' color='primary' disabled={!jobInfo} onClick={sendToBg}>Send to background</Button>
```

Comment:
I think we need something other than this check for timing the buttons' disabling more accurately, because of the bug I mentioned above.